samples/msal-angular-v3-samples/angular-b2c-sample-app/README.md

```diff
@@ -49,4 +47,4 @@ Implementing B2C user-flows is a matter of initiating authorization requests aga
 
 ## Common errors
 
-- If your app is running into redirect loops when trying to acquire a token for a resource such as your web API, make sure you have granted **admin consent** to the permissions/scopes required for that resource on App registration portal. See [Using redirects in MSAL Angular v2](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/redirects.md) for more on redirect experience. See [MSAL Angular v2 Errors](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/errors.md) for other common errors.
\ No newline at end of file
+- If your app is running into redirect loops when trying to acquire a token for a resource such as your web API, make sure you have granted **admin consent** to the permissions/scopes required for that resource on App registration portal. See [Using redirects in MSAL Angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/redirects.md) for more on redirect experience. See [MSAL Angular Errors](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/errors.md) for other common errors.
```

Comment:
Note: Links to docs/v2-docs will be updated in a later PR.

Reply:
Suggest having a task in ADO to make sure we don't forget